### PR TITLE
noscript, uncomment config, change sw scope

### PR DESCRIPTION
**Pull request by xxyzz**

### CODE REVIEW

**Comment by Anduin2017** [comment]
Well done! I will check it later.

File: js/AiurUI.js [hunk 1]
```diff
@@ -37,23 +34,24 @@ var countDownDate = new Date("Dec 7, 2017 13:00:00").getTime();
...
+    if (distance < 0) {
+        clearInterval(x);
+        $("#started").html("选课已经开始");
+        return;
+    }
     var days = Math.floor(distance / (1000 * 60 * 60 * 24));
     var hours = Math.floor((distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
     var minutes = Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60));
     var seconds = Math.floor((distance % (1000 * 60)) / 1000);
     document.getElementById("demo").innerHTML = days + "天 " + hours + "小时 "
```
Comment (Anduin2017) [inline js/AiurUI.js]: Please check here if it can be replaced with jQuery like method?

File: index.html [hunk 2]
```diff
@@ -71,36 +73,41 @@ <h1 class="loading">选课平台
                     <span class="d d-3">.</span>
                 </h1>
                 <p>选课活动开始，请在选课前仔细阅读此说明。</p>
+                <noscript>
```
Comment (Anduin2017) [inline index.html:76]: Is that your idea to show a warning here?

**Review by Anduin2017 — COMMENTED** [review]
Your changes are very reasonable. But there still remains some issues.

We will provide an API about when the scheduled course selection starts. For further features, we may reconstruct the entire project. Thanks for your patience and collaborating!.

**Comment by Anduin2017** [comment]
By the way, next time you create any pull request, please create some description to describe the following items:

- What is the motivation for the pull request
- What is the difference from before?

File: index.html (hunk 2 above)
Comment (xxyzz) [reply index.html:76]: It will show when browser disable JavaScript, in case some users can't see the alert.

File: js/AiurUI.js [hunk 3]
```diff
@@ -36,7 +36,7 @@ var update = function () {
     var distance = countDownDate - now;
     if (distance < 0) {
         clearInterval(x);
-        $("#started").html("选课已经开始");
+        document.getElementById("started").textContent = "选课已经开始";
```
Comment (Anduin2017) [inline js/AiurUI.js]: Oh, I mean, replace line 46 with jQuery selector. For running here we already have jQuery accessible.

**Review by Anduin2017 — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (Anduin2017) [reply index.html:76]: Nice. that was very reasonable.

**Review by Anduin2017 — APPROVED** [review]

**Pull request merged**